tensorflow/core/common_runtime/dml/dml_util.cc

```diff
@@ -201,17 +201,40 @@ DmlTensorLayout GetDmlTensorLayout(TensorFormat format, uint32_t rank) {
   return tensor_layout;
 }
 
-dml::TensorLayout GetDmlXTensorLayout(TensorFormat format) {
+dml::TensorPolicy GetDmlXTensorPolicy(TensorFormat format) {
   switch (format) {
     case FORMAT_NHWC:
-      return dml::TensorLayout::Nhwc;
+      return dml::TensorPolicy::InterleavedChannel();
```

Comment:
Tensorflow uses "channels_first" to name NCHW and "channels_last" to name NHWC. Personally, I think it's more descriptive than "InterleavedChannel", and it follows the existing tensorflow conventions.

Reply:
I've never much liked "channels first" (because they really aren't). "Interleaved" is used distinct from "Planar", which is terminology from the graphics world. "Default" and "ChannelsLast" might be reasonable, though. I'll have to think about it...